api/tests/integration/domain/usecases/start-campaign-participation_test.js

```diff
@@ -110,5 +150,35 @@ describe('Integration | UseCases | start-campaign-participation', function() {
       const assessments = await knex('assessments');
       expect(assessments).to.have.lengthOf(0);
     });
+
+    context('when campaign is multipleSendings', function() {
+      beforeEach(async function() {
+        campaignId = databaseBuilder.factory.buildCampaign({ multipleSendings: true, idPixLabel: 'identifiant', organizationId, targetProfileId }).id;
+        await databaseBuilder.commit();
+      });
+
+      context('when it is its first participation', function() {
```

Comment:
**question :** C'est quasiment les mêmes tests que pour le contexte 'when campaign has idPixLabel and no participantExternalId' ? On a besoin de faire la distinction ?